utest/model/test_keyword.py

```diff
@@ -33,6 +34,22 @@ def test_id_with_keyword_parents(self):
     def test_slots(self):
         assert_raises(AttributeError, setattr, Keyword(), 'attr', 'value')
 
+    def test_copy_keyword(self):
+        kw = Keyword()
+        kw_new = copy.copy(kw)
+        self.assertEqual(kw.name, kw_new.name)
+
+        kw_new.name = kw.name + '1'
+        self.assertNotEqual(kw.name, kw_new.name)
+
+        self.assertEqual(id(kw.tags), id(kw_new.tags))
```

Comment:
Good tests. Two nitpick comments:

- We have our own `assert_equal` and friends.
- We don't use empty lines inside functions (unless functions are long but we also avoid long functions).